thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -64,20 +73,23 @@
   public static final String DEFAULT_ALERT_RECEIVER = DefaultDetectionOnboardJob.DEFAULT_ALERT_RECEIVER_ADDRESS;
 
   public static final Boolean DEFAULT_IS_ACTIVE = true;
-  public static final String DEFAULT_METRIC_FUNCTION = "SUM";
   public static final String DEFAULT_WINDOW_DELAY = "0";
   public static final String DEFAULT_ALERT_CRON = "0 0/5 * 1/1 * ? *"; // Every 5 min
+  public static final String DEFAULT_ALERT_FILTER_PATTERN = AlertFilterAutoTuneOnboardingTask.DEFAULT_AUTOTUNE_PATTERN;
+  public static final String DEFAULT_ALERT_FILTER_SENSITIVITY_LEVEL = AlertFilterAutoTuneOnboardingTask.DEFAULT_AUTOTUNE_SENSITIVITY_LEVEL;
```

Comment:
As the latest design, we don't need to take SENSITIVITY_LEVEL anymore.  Please change this to AUTOTUNE_TYPE, AUTOTUNE_FEATURES; AUTOTUNE_MTTD Thanks

Reply:
Can you give me the field name of these features?

Reply:
AUTOTUNE_TYPE = "autotuneType"; AUTOTUNE_TYPE_VAL = "AUTO_TUNE"

Reply:
Fixed.